### DISCUSSION TITLE
is   redis/go-version is a new version for  go-redis/redis

### DISCUSSION THREAD

**Discussion by begemot63** [redis/go-redis#2560] [Q&A]
what are differences between redis/go-version and  go-redis/redis ?
And why https://pkg.go.dev/github.com/go-redis/redis/v9  link to repository points to https://github.com/redis/go-redis?

What to use in go.mod  If I want to upgrade from  github.com/go-redis/redis v6.15.5+incompatible

**Comment by monkey92t** [comment]
v9: 
repo: https://github.com/redis/go-redis
go mod: github.com/redis/go-redis

before v9(v8 and earlier):
repo: https://github.com/go-redis/redis
go mod: github.com/go-redis/redis

redis/go-redis is migrated from go-redis/redis. If you visit go-redis/redis, you will be automatically redirected to redis/go-redis. However, in go-mod, versions v8 or v7 of go-redis/redis are still available.

**Reply by zhangzhuang15** [reply]
>  If you visit go-redis/redis, you will be automatically redirected to redis/go-redis

I'm just curious.

how to do this redirection? Does Github provide some tools ?
